File: app/src/main/java/com/doapps/habits/viewholder/ProgramViewHolder.java
package com.doapps.habits.viewholder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.doapps.habits.R;

public class ProgramViewHolder extends RecyclerView.ViewHolder {
    private final TextView txtTitle;
    private final TextView txtPercent;

    public ProgramViewHolder(final View itemView) {
        super(itemView);
        txtTitle = (TextView) itemView.findViewById(R.id.program_title);
        txtPercent = (TextView) itemView.findViewById(R.id.program_percent);
    }

    public TextView getTitleTextView() {
        return txtTitle;
    }

    public TextView getPercentTextView() {
        return txtPercent;
    }
}

File: app/src/main/java/com/doapps/habits/view/holders/ProgramViewHolder.kt
package com.doapps.habits.view.holders

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView

import com.doapps.habits.R

class ProgramViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    val titleTextView: TextView = itemView.findViewById(R.id.program_title)
    val percentTextView: TextView = itemView.findViewById(R.id.program_percent)
}
